### DISCUSSION TITLE
v5.2.2 Publication failed

### DISCUSSION THREAD

**Discussion by ceady-tw** [Onemind-Services-LLC/react-icons-ng#542] [General]
It looks like the v5.2.2 publication to the repository failed. Sorry if this is considered a duplicate thread. Opening it since my other one was marked as closed. 😕 

https://github.com/Onemind-Services-LLC/react-icons-ng/actions/runs/10151363064

**Comment by abhi1693** [comment]
I have published the package manually and will check if I can fix the auto-release ci.

**Reply by ceady-tw** [reply]
Thank you! 🙏
